public/app/plugins/panel/barchart/BarChartPanel.tsx

```diff
@@ -305,6 +308,8 @@ export const BarChartPanel = ({ data, options, fieldConfig, width, height, timeZ
     });
   };
 
+  const showNewVizTooltips = config.featureToggles.newVizTooltips;
```

Comment:
this is consistent with the other panels. previously there were more conditions here, so the name change was a little less bizzare. this will all go away soon anyways, and keeping it consistent is easier for search/replace.